src/libraries/Common/src/System/Security/Cryptography/SP800108HmacCounterKdf.cs

```diff
@@ -194,10 +194,10 @@ public static byte[] DeriveBytes(byte[] key, HashAlgorithmName hashAlgorithm, by
         ///   <paramref name="hashAlgorithm"/> is not a known or supported hash algorithm.
         /// </exception>
         /// <exception cref="EncoderFallbackException">
-        ///   <paramref name="label" /> or <paramref name="context" /> contains text that cannot be converted to UTF8.
```

Comment:
Just a reminder that a similar change will be needed to the actual docs repo if we want it to appear in official docs.

Reply:
I may or may not have a follow up for that at some point. If not then in the happy rainbow times when docs are built from these comments, it will fix itself.